docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: onsite/components-react
```

Comment:
vc n tá usando o build acima? pra que `image`?

Reply:
sempre coloco pra facilitar quando for remover imagens do docker do pc